doc/api_assets/style.css

```diff
@@ -330,6 +330,21 @@ hr {
   margin-top: .666em;
 }
 
+#toc .stability_0::after {
+  background-color: #d50027;
+  color: #fff;
+}
+
+#toc .stability_0::after {
+  content: "deprecated";
+  font-size: .8em;
+  position: relative;
+  top: -.1em;
```

Comment:
Not sure why (em cascade?), but `-.18em` seems to center it better:

![](https://cloud.githubusercontent.com/assets/115237/19246566/49962282-8f27-11e6-9236-c48ba5d73631.png)